### DISCUSSION TITLE
How do you fix "Missing module docstringpylint(missing-module-docstring)"

### DISCUSSION THREAD

**Discussion by smartman0307** [community/community#50660] [Actions]
### Select Topic Area

Question

### Body

I'm using the pygame module on VS Code and I ran into the issue where the pygame has not init member. I followed the solutions to [this](https://stackoverflow.com/questions/50569453/why-does-it-say-that-module-pygame-has-no-init-member) link. I edited the user settings and added

"python.linting.pylintArgs": [
    "--extension-pkg-whitelist=pygame",
    "--unsafe-load-any-extension=y"
]
to the end of the json file

The pygame problem was resolved. However, when I use import random. I get this warning:

Missing module docstringpylint(missing-module-docstring)

How do I make it go away? Also, is there a better way to resolve the init problem of pygame?

**Comment by SuperJUNOed** [comment]
I just figured out what docstrings are. They just describe the function or class. It's enclosed in three double quotation marks or single quotation marks. This helped me.

To remove docstring warnings in VSCode, I just added "--disable=C0111" to "python.linting.pylintArgs": [], which was in the User's JSON settings.

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
